be/test/olap/tablet_mgr_test.cpp

```diff
@@ -209,6 +209,21 @@ TEST_F(TabletMgrTest, DropTablet) {
     ASSERT_TRUE(!dir_exist);
 }
 
+TEST_F(TabletMgrTest, GetRowsetId) {
+    std::string path = _engine_data_path + "/data/0/15007/368169781/020000000000000100000000000000020000000000000003_0_0.dat";
+    TTabletId tid;
+    TSchemaHash schema_hash;
+    ASSERT_TRUE(_tablet_mgr.get_tablet_id_and_schema_hash_from_path(path, &tid, &schema_hash));
+    ASSERT_EQ(15007, tid);
+    ASSERT_EQ(368169781, schema_hash);
```

Comment:
should add some bad case such as the path is not a rowset path.

Reply:
OK